state/status_history_test.go

```diff
@@ -142,6 +142,22 @@ func (s *StatusHistorySuite) TestPruneStatusHistoryByDate(c *gc.C) {
 	}
 }
 
+func (s *StatusHistorySuite) TestStatusHistoryFilterRunningUpdateStatusHook(c *gc.C) {
```

Comment:
how does this test pass?
there's no filter for update-status hook
the test should really be TestStatusHistoryExcludeFilter()
also, we want a test where there are no excludes